src/services/findAllReferences.ts

```diff
@@ -1523,9 +1523,14 @@ namespace ts.FindAllReferences {
             }
 
             // Use the parent symbol if the location is commonjs require syntax on javascript files only.
-            referenceSymbol = isInJSFile(referenceLocation) && referenceLocation.parent.kind === SyntaxKind.BindingElement && isRequireVariableDeclaration(referenceLocation.parent)
-                ? referenceLocation.parent.symbol
-                : referenceSymbol;
+            if (isInJSFile(referenceLocation)
+                && referenceLocation.parent.kind === SyntaxKind.BindingElement
+                && isRequireVariableDeclaration(referenceLocation.parent)) {
+                referenceSymbol = referenceLocation.parent.symbol;
+                // The parent will not have a symbol if it's an ObjectBindingPattern (when destruturing is used).  In

```

Comment:
Please fix the typo (destruturing) so we don't get a PR from someone else later 🙂

Reply:
Bah!  Sorry -- fixed.  (Dusty keyboard...)